docs/lib/storage/fragments/ios/configureaccess.md

```diff
@@ -1,19 +1,26 @@
 ## Protected access
 
-Create an options object with the protected access level to restrict access for certain objects.
+Create an `Options` object with the `.protected` access level to restrict access for your key.
 
 ```swift
-let options = StorageDownloadDataRequest.Options(accessLevel: .protected)
-Amplify.Storage.downloadData(key: "myKey", options: options) { (event) in
+let dataString = "My Data"
+let data = dataString.data(using: .utf8)!
+let options = StorageUploadDataRequest.Options(accessLevel: .protected)
+Amplify.Storage.uploadData(key: "userProtectedKey", data: data, options: options) { (event) in
     // handle result
 }
 ```
 
-When uploading a file with `protected` access level, users can only read the file and only the user which created the file can delete it.
+When uploading a file with `.protected` access level:
+* Owners have the ability to read, write, and delete access
+* Other users only have read access
 
-Another user that wants to read the file can specify the user that created it:
+If another user wants to read the file, you will need to include the `targetIdentityId` parameter:
```

Comment:
Above it says other users only have read access, but this states users need to be included in the `targetIdentifyId` parameter to gain access.. I'm a bit confused. "If another user wants to read the file" -- does this mean to say if I want to grant read access to a specific user?

Reply:
Chatted offline: need to update to be more clear about if user1 uploads a protected file, then user2 can access this file in the following manner.